Yoast/wordpress-seo
# Improve the featured image warning

## Summary

Improve the styling of the featured image warning and send an audible message for screen readers.

## Relevant technical choices:

- uses native WP notices with the "alternative" styling, see screenshot below
- inserts the warning inside the box
- adds "SEO issue" to the warning string, to let users know Yoast SEO is throwing the warning
- draws a border around the image instead of the whole box
- uses `a11y-speak`
- a few, minor, JS optimizations

## Test instructions

- build CSS and JS
- edit a post and insert a featured image smaller then 200 x 200
- check the warning and border appear correctly
- check the `a11y-speak` message gets correctly inserted in the ARIA live region
- remove the image (or insert an image greater then 200 x 200) and check the warning and borders disappear correctly

Fixes #6275 

<img width="321" alt="screen shot 2017-01-19 at 13 57 12" src="https://cloud.githubusercontent.com/assets/1682452/22108786/10c25c0a-de55-11e6-96c5-6c493d6418ad.png">
